admin/src/components/Contract.js

```diff
@@ -657,24 +670,24 @@ export default ({ young, admin }) => {
                 <div style={{ display: "flex", justifyContent: "center" }}>
                   <LoadingButton
                     onClick={async () => {
-                      const erroredFields = await validateForm();
-                      if (Object.keys(erroredFields).length) toastr.error("Il y a des erreurs dans le formulaire");
                       if (contract?.invitationSent === "true") {
                         const confirmText =
                           "Si vous enregistrez les modifications, les parties prenantes ayant validé recevront une notification et devront à nouveau valider le contrat d'engagment. De la même manière, les parties prenantes dont l'email a été modifié recevront également un email.";
                         if (confirm(confirmText)) {
+                          const erroredFields = await validateForm();
+                          if (Object.keys(erroredFields).length) toastr.error("Il y a des erreurs dans le formulaire");
```

Comment:
```suggestion
                          if (Object.keys(erroredFields).length) return toastr.error("Il y a des erreurs dans le formulaire");
```

il y a le meme changmeent a faire ligne 698